gov/governance/config.gno

```diff
@@ -17,40 +17,17 @@ var (
 )
 
 func init() {
-	// value for testing case from @notJoon
+	// https: //docs.gnoswap.io/core-concepts/governance
 	config = Config{
-		// XXX: TODO, REMOVE AFTER QA
-		// - config for testcase
-		VotingStartDelay:              uint64(10),   // 10s ≈ 5 block
-		VotingPeriod:                  uint64(30),   // 30s ≈ 15 block
-		VotingWeightSmoothingDuration: uint64(10),   // 10s ≈ 5 block
-		Quorum:                        uint64(50),   // 50% of total xGNS supply
-		ProposalCreationThreshold:     uint64(100),  //
-		ExecutionDelay:                uint64(10),   // 10s ≈ 5 block
-		ExecutionWindow:               uint64(1000), // 500 block
-
-		// XXX: TODO, REMOVE AFTER QA
-		// - config for live qa
-		// VotingStartDelay:              uint64(180),           // 3m
-		// VotingPeriod:                  uint64(600),           // 10m
-		// VotingWeightSmoothingDuration: uint64(600),           // 10m
-		// Quorum:                        uint64(50),            // 50% of total xGNS supply
-		// ProposalCreationThreshold:     uint64(1_000_000_000), // 1_000_000_000
-		// ExecutionDelay:                uint64(60),            // 1m
-		// ExecutionWindow:               uint64(3600),          // 1h
+		VotingStartDelay:              uint64(86400),         // 1d
```

Comment:
as i know, dongwon have request to use governanve config for QA. need to check it.

Reply:
yes, let's set all configs with QA time for now, and when we're done with the final QA before `beta v3`, we can roll all back.

Reply:
FYI, current dev chain deployed with qa timing

Reply:
9bb9416667c213fff1860a24ff3686b9aaef4c69